### PR TITLE
Fix pickling issue

### PR DESCRIPTION
**Pull request by yalef**
Resolves: #31 #30

### CODE REVIEW

**Review by TheSuperiorStanislav — COMMENTED** [review]
Could add a test to verify that it works?

**Comment by yalef** [comment]
> Could you add a test to verify that it works?

I've added test for issue with pickle error (occurs when ValidationError related to field is raised during import)

But I won't add test for `AttributeError` case, because in this case error occur just because `Job.result` field had `""` default value

File: tests/fake_app/migrations/0002_alter_artist_name.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.7 on 2024-05-17 05:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (TheSuperiorStanislav) [inline tests/fake_app/migrations/0002_alter_artist_name.py:6]: Can't we just re-create init migration for fake app?

**Review by TheSuperiorStanislav — APPROVED** [review]

File: import_export_extensions/results.py [hunk 2]
```diff
@@ -23,10 +26,57 @@ def __reduce__(self):
...
+
+        Otherwise, we will go through all nested validation errors
+        and build new `ValidationError` with only one level of
+        nested `ValidationError` instances.
+
+        """
+        result = collections.defaultdict(list)
+        if not hasattr(value, "message_dict"):
+            self._validation_error = value
+            return
```
Comment (NikAzanov) [inline import_export_extensions/results.py:72]:
```suggestion
        if not hasattr(value, "message_dict"):
            self._validation_error = value
            return
        result = collections.defaultdict(list)
```

**Review by NikAzanov — APPROVED** [review]

**Pull request merged**